scripts/docker/install_airflow.sh

```diff
@@ -56,26 +56,26 @@ function install_airflow() {
             pip uninstall apache-airflow --yes
             pip install ${AIRFLOW_INSTALL_EDITABLE_FLAG} \
                 "${AIRFLOW_INSTALLATION_METHOD}[${AIRFLOW_EXTRAS}]${AIRFLOW_VERSION_SPECIFICATION}"
-        fi
+            fi
```

Comment:
This seems unintended?

```suggestion
        fi
```

Reply:
Right